src/components/card/card.stories.tsx

```diff
@@ -15,12 +15,24 @@ import VerticalDivider from "../vertical-divider";
 import IconButton from "../icon-button";
 
 export const DefaultStory: ComponentStory<typeof Card> = (args: CardProps) => (
-  <Card {...args}>
+  <Card
+    {...args}
+    onClick={() => {}}
+    // footer={
+    //   <CardFooter>
+    //     <CardColumn>
+    //       <Link icon="link" href="https://carbon.sage.com/">
+    //         Footer link
+    //       </Link>
+    //     </CardColumn>
+    //   </CardFooter>
+    // }
```

Comment:
**issue**: I assume we would want this commented code instead of lines 51-57 given the tests now we have?